client/app/scripts/superdesk/ui/views/sd-timepicker-alt.html

```diff
@@ -1,8 +1,23 @@
-hours:
-<select ng-model="hours" ng-change="update()">
-    <option ng-repeat="h in range(0, 23)">{{h}}</option>
-</select>
-minutes:
-<select ng-model="minutes" ng-change="update()">
-    <option ng-repeat="m in range(0, 59, step)">{{m}}</option>
-</select>
\ No newline at end of file
+<div>
+    <span class="input-small">{{hours|leadingZero}}:{{minutes|leadingZero}}</span>
+    <button class="btn btn-default" ng-click="open = !open"><i class="icon-time"></i></button>
+</div>
+
+<div class="timepicker-popup" ng-show="open" ng-keydown="keydown({e: $event})" tabindex="0">
+    <div class="timepicker-core">
+        <div class="select-area">
+            <div class="header" translate>Hours</div>
+            <ul>
+                <li ng-repeat="h in range(0, 23)" ng-click="$parent.hours = h" ng-class="{active: hours === h}">{{h | leadingZero}}</li>
```

Comment:
it would be good to generate those ranges in directive, otherwise it will create new list every $digest